prisma-fmt/src/get_config.rs

```diff
@@ -31,35 +32,36 @@ pub(crate) fn get_config(params: &str) -> Result<String, String> {
     get_config_impl(params)
         .map_err(|err| {
             json!({
-                "error": {
-                    "message": err.message,
-                    "error_code": err.error_code,
-                }
+                "message": err.message,
+                "error_code": err.error_code,
             })
             .to_string()
         })
         .map(|value| value.to_string())
 }
 
 fn get_config_impl(params: GetConfigParams) -> Result<serde_json::Value, GetConfigError> {
-    let mut config = match psl::parse_configuration(&params.prisma_schema) {
-        Ok(config) => config,
-        Err(err) => {
-            return Err(GetConfigError {
-                error_code: None,
-                message: err.to_pretty_string("schema.prisma", &params.prisma_schema),
-            })
+    let wrap_get_config_err = |errors: Diagnostics| -> GetConfigError {
+        use std::fmt::Write as _;
+
+        let mut full_error = errors.to_pretty_string("schema.prisma", &params.prisma_schema);
+        write!(full_error, "\nValidation Error Count: {}", errors.errors().len()).unwrap();
+
+        let parser_error = user_facing_errors::common::SchemaParserError { full_error };
+
+        GetConfigError {
+            error_code: None,
```

Comment:
This should actually be `Some(1012)`, from https://github.com/prisma/prisma-engines/blob/a3623c31d4eb22f2dccd198738b449488c9b55dd/libs/user-facing-errors/src/common.rs#L182-L186.

@tomhoule, any help on how to set that error_code programmatically (without a brutal copy-paste)? I can't select the `ERROR_CODE` property of `parser_error` as that's set by a procedural macro

Reply:
Solved on [slack](https://prisma-company.slack.com/archives/C02A1646TBR/p1666101752437719).